test/users.spec.ts

```diff
@@ -122,18 +122,6 @@ describe('Users', function () {
     });
   });
 
-  /**
-   * remove totp
-   */
-
-  it('should remove totp', async () => {
```

Comment:
is there no way to remove `totp` anymore in keycloak 11?

Reply:
It seems it was moved to a general 'credentials'-resource including both OTP and WebAuthn. [1]

We should probably just disable the test for Keycloak 8+ instead of removing the test. I think Github Actions sets some kind of environment variable during the matrix build that can be used to skip tests.

[1] https://www.keycloak.org/docs/latest/release_notes/#support-for-password-less-authentication-multi-factor-authentication-and-multiple-credentials-per-user

Reply:
It's not possible using the users API. You have to use the credentials endpoint as described in the commit message.

Reply:
I see you skip the test with an if-clause. Mocha has support for runtime test skipping with `this.skip()`[1]. This will include the test in the ouput as 'pending'. 

I don't mind either way. @edewit - do you have any further comments?

[1] https://mochajs.org/#inclusive-tests

Reply:
Nice, thank you for the pointer. I'll use `this.skip()`.